lib/chore/manager.rb

```diff
@@ -5,14 +5,15 @@
 module Chore
   # Manages the interactions between fetching messages (Consumer Strategy), and working over them (Worker Strategy)
   class Manager
+    include Util
 
     def initialize()
       Chore.logger.info "Booting Chore #{Chore::VERSION}"
       Chore.logger.debug { Chore.config.inspect }
+      procline("chore-master-#{Chore::VERSION}:Started:#{Time.now}")
```

Comment:
This will mean a change to some of our monitoring, as we look for a specific procline to determine the master.

That said, highly approve of this change.